test/integration/guest_can_change_quanity_in_cart_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class GuestCanChangeQuanityInCartTest < ActionDispatch::IntegrationTest
+  test "guest visits cart and adds more items" do
+    create_and_add_item_to_cart
```

Comment:
Yes, I was going to extract this before I merge with master. We use in the guest can remove from cart test.